### PR TITLE
Added: TransferFilterParams for Searching Transfer feature

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

File: OmiseSwift/CurrencyDataConverter.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+
+public class CurrencyDataConverter: Converter {
```
Comment (chakrit) [inline OmiseSwift/CurrencyDataConverter.swift]: Just `CurrencyConverter` suffice?

**Review by chakrit — CHANGES_REQUESTED** [review]

File: OmiseSwift/CurrencyDataConverter.swift (hunk 1 above)
Comment (pitiphong-p) [reply OmiseSwift/CurrencyDataConverter.swift]: I thought about this name at first and then I realized that `Will the API consumer think that this class is a currency converter which convert money between currencies?` So I think it should be named with other name

Comment (chakrit) [reply OmiseSwift/CurrencyDataConverter.swift]: `Data` is still pretty generic na. Maybe `CurrencyFieldConverter`

Comment (pitiphong-p) [reply OmiseSwift/CurrencyDataConverter.swift]: Umm SGTM

Comment (pitiphong-p) [reply OmiseSwift/CurrencyDataConverter.swift]: Done :)

**Review by chakrit — APPROVED** [review]

**Pull request merged**